scheduler/generic_sched.go

```diff
@@ -724,6 +724,13 @@ func (s *GenericScheduler) computePlacements(destructive, place []placementResul
 				if stopPrevAlloc {
 					s.plan.PopUpdate(prevAllocation)
 				}
+
+				if prevAllocation != nil {
+					if missing.IsRescheduling() {
+						annotateRescheduleTracker(prevAllocation, now)
+					}
+				}
```

Comment:
am I right that the main behavior change stems from here?  I think it could benefit from a code comment akin to the ones above.